misc-scripts/gifts_import/populate_protein_feature.pl

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/env perl
```

Comment:
A little more inline documentation overall please, just so whoever has to take this up in a few years has a better understanding. It's all pretty self evident, but some things like the SELECT/INSERT, why some values are hardcoded, etc just to make life easier.